packages/wonder-blocks-clickable/components/clickable-behavior.js

```diff
@@ -96,6 +96,9 @@ type Props = {|
      *
      * If both safeWithNav and beforeNav are provided, beforeNav will be run
      * first and safeWithNav will only be run if beforeNav does not reject.
+     *
+     * WARNING: Using this with `target="_blank"` will trigger built-in popup
+     * blockers in Firefox and Safari.
```

Comment:
I dunno, this feels like a huge issue - big enough that I think we should prevent people from doing it! We could at least change the Flow types to make it mutually exclusive. And then update the docs to make it clear that `beforeNav` is impossible to use with target=_blank and that folks should instead use safeWithNav.

Reply:
Normally I would immediately jump to enforce this with flow, but the props are already pretty complicated with an existing disjoint union.  I'll give it a try and see how hard it is to make it work.